### PR TITLE
 Community Article - Apache Will Not Restart: INFODEV-3265

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-c2bb61fd91/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-5310440e3e/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b06a9a931e/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-63306f54b1/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
@RackerKate On it.

**Comment by cyrichardson** [comment]
@RackerKate Suggested changes here: 
[restart-apache_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2783652/restart-apache_cyrEdits.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-45dcc3e3c3/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e88ff001ea/how-to/)

File: content/cloud-servers/restart-apache.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    Syntax error on line 51 of /etc/httpd/conf/httpd.conf:
+    Invalid command 'erverRoot', perhaps misspelled or defined by a module not included in the server configuration
+
+### Check the Apache error logs
+
+If you resolve those errors and Apache still doesn't restart, check the Apache error logs. Using two windows might be helpful. In one window, use the tail command against the error log by running the following command:
+
+    tail –f /var/log/httpd/error _log
+
```
Comment (chri2547) [inline content/cloud-servers/restart-apache.md]: I'm thinking maybe the end italics is missing or is incorrect?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b647d7450c/how-to/)

**Pull request merged**